src/components/userform.js

```diff
@@ -142,6 +142,18 @@ class UserForm extends Component {
       // 'position':'absolute',
       // 'margin-top':'0'
     };
+    try {
```

Comment:
I think we need this mixpanel track, please check

Reply:
Have Added the mixed panel